bin/improver-temp-lapse-rate

```diff
@@ -94,14 +101,22 @@ def main():
     orography_cube = load_cube(args.orography_filepath)
     land_sea_mask_cube = load_cube(args.land_sea_mask_filepath)
 
-    result = LapseRate(
-        max_height_diff=args.max_height_diff,
-        nbhood_radius=args.nbhood_radius,
-        max_lapse_rate=args.max_lapse_rate,
-        min_lapse_rate=args.min_lapse_rate).process(temperature_cube,
-                                                    orography_cube,
-                                                    land_sea_mask_cube)
-
+    if args.return_dalr:
+        result = temperature_cube.copy(
+            data=np.full_like(temperature_cube.data, U_DALR.points[0]))
+        result.rename('lapse_rate')
```

Comment:
If changing cube name (see comment in plugin) then change it here too.

Reply:
Updated to use "air_temperature_lapse_rate".